branding/branding

```diff
@@ -2,14 +2,14 @@ ISO_PV_TOOLS_LABEL=XCP-ng Tools
 PRODUCT_BRAND=XCP-ng
 BRAND_GUEST_SHORT=VM
 COMPANY_NAME=Open Source
-PRODUCT_VERSION_TEXT_SHORT=7.4
+PRODUCT_VERSION_TEXT_SHORT=7.5
```

Comment:
I don't like it that we must change the values in this file manually but that will be OK for 7.5. In the future we should replace these with placeholders that would be replaced during build of the xenserver-release (or xcp-ng-release once renamed) package.